cmd/overworker/main.go

```diff
@@ -16,7 +17,7 @@ import (
 func main() {
 	rand.Seed(time.Now().UTC().UnixNano())
 
-	cfg := worker.NewDefaultConfig()
+	cfg := workerConfig.NewDefaultConfig()
```

Comment:
The main reason: I couldn't import it in room.go because:
```
import cycle not allowed
package github.com/giongto35/cloud-game/cmd/overworker
        imports github.com/giongto35/cloud-game/pkg/worker
        imports github.com/giongto35/cloud-game/pkg/worker/room
        imports github.com/giongto35/cloud-game/pkg/worker
```
Is it possible to fix this somehow?

Reply:
yeah. I understand the scenario. There is two way of solving this - 1. that what you did, 2. Move it to a root directory maybe something similar to - `configs/worker`. 
I don't mind having any approach. But let's keep the alias name in all small characters.

Reply:
Yes, I think it's better to have config in root directory: config/worker, so we can have config easily sharable and accessible between package. It also reduces all the dependencies.

Reply:
and as sadlil mentioned, in Golang package should be all lower case.